libraries/AP_Camera/examples/tracking.py

```diff
@@ -0,0 +1,131 @@
+"""
+Required Installations:
+pip install opencv-python-headless pymavlink ultralytics
+sudo apt-get install libgstreamer1.0-0 gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-good
+"""
+
+import cv2
+import threading
+from pymavlink import mavutil
+import math 
+from ultralytics import YOLO
+import argparse
+
+class DroneGimbalControl:
+    def __init__(self, video_source=None):
+        # Set default video source if not provided
+        if video_source is None:
+            video_source = 'udpsrc port=5600 ! application/x-rtp,encoding-name=H264,payload=96 ! rtph264depay ! h264parse ! queue ! avdec_h264 ! videoconvert ! appsink'
```

Comment:
I thought that with the siyi cameras, only H265 worked.  I don't think H264 streaming works with them